{"source": "exploit-database", "cves": ["CVE-2015-5273", "CVE-2015-5287"], "cpes": {"cpe:/o:redhat:enterprise_linux:7.0:ga:server": {"scores": {"stride": "000000"}}}, "filename": "platforms/multiple/local/38835.py"}